knowledge_graph/routes.py

```diff
@@ -168,75 +167,105 @@ def receive_user_scores() -> Tuple[Response, int]:
     except KeyError:
         return make_response("invalid key"), 400
 
-    response = {"session-id": session_id}
+    response = {"sessionId": session_id}
 
-    response = Response(dumps(response))
+    response = jsonify(response)
     return response, 201
 
 
 @app.route("/personal_values", methods=["GET"])
+@auto.doc()
 def get_personal_values():
-    """Given a session-id, this returns the top three personal values for a user"""
+    """
+    Returns the top 3 personal values of a user given a session ID.
+    """
     try:
         session_id = str(request.args.get("session-id"))
-    except:
+    # TODO: catch exceptions properly here
+    except Exception:
         return make_response("Invalid Session ID Format or No ID Provided"), 400
 
-    scores = db.session.query(Scores).filter_by(session_id=session_id).first()
+    scores = Scores.query.filter_by(session_id=session_id).first()
     if scores:
         scores = scores.__dict__
-        del scores["_sa_instance_state"]
-        del scores["session_id"]
+        personal_values_categories = [
+            "security",
+            "conformity",
+            "benevolence",
+            "tradition",
+            "universalism",
+            "self_direction",
+            "stimulation",
+            "hedonism",
+            "achievement",
+            "power",
+        ]
+        sorted_scores = {key: scores[key] for key in personal_values_categories}
+        # del scores["_sa_instance_state"]
+        # del scores["session_id"]
+        # del scores["user_id"]
+        # del scores["scores_id"]
+
+        top_scores = sorted(sorted_scores, key=sorted_scores.get, reverse=True)[:3]
 
-        top_scores = sorted(scores, key=scores.get, reverse=True)[:3]
         try:
-            with open("value_descriptions.json", "r") as f:
+            file = os.path.join(os.getcwd(), "json_files/value_descriptions.json")
```

Comment:
this should also be:
`file = os.path.join(os.getcwd(), "json_files","value_descriptions.json")`